File: 02 Datentypen/Go/Aufgabe6.go
package main

import "fmt"

var eingabe float32

func main() {
	fmt.Print("Bitte gib eine Zahl ein: ")
	fmt.Scan(&eingabe)

	fmt.Println() // Leere Zeile ausgeben, fmt.Print('\n'); geht auch.

	fmt.Println("Deine eingegebene Zahl ist ", eingabe, ".")
}

File: 02 Datentypen/Pascal/Aufgabe6.pas
Program Aufgabe6;

{$codepage utf8}

Var Eingabe: real;

Begin
    Write('Bitte gib eine Zahl ein: ');
    ReadLn(Eingabe);
    WriteLn(); // Leere Zeile ausgeben, Write('\n'); geht auch.

    WriteLn('Deine eingegeben Zahl ist ', Eingabe, '.');
End.
